Surface_mesh_simplification/doc/Surface_mesh_simplification/PackageDescription.txt

```diff
@@ -3,12 +3,14 @@
 /// \ingroup PkgSurfaceMeshSimplificationRef
 /*!
 \addtogroup PkgSurfaceMeshSimplificationRef
-\todo check generated documentation
 \cgalPkgDescriptionBegin{Triangulated Surface Mesh Simplification,PkgSurfaceMeshSimplification}
 \cgalPkgPicture{SMS-detail.png}
 \cgalPkgSummaryBegin
-\cgalPkgAuthor{Fernando Cacciola}
-\cgalPkgDesc{This package provides an algorithm to simplify a triangulated surface mesh  by edge collapsing.  It is an implementation of the Turk/Lindstrom <I>memoryless</I>  surface mesh simplification algorithm.}
+\cgalPkgAuthor{Fernando Cacciola, Mael Rouxel-Labbé, and Baskın Şenbaşlar}
```

Comment:
Ş is generating a doc warning:
`WARNING: a non ascii character has been found in author string for bibtex (probably a non-handled accentuated letter).Check the new package added and update the function protect_accentuated_letters in Documentation/scripts/generate_how_to_cite.py`

You need to update this python script to handle the lower and upper case latter.